app/src/main/res/values/dimens.xml

```diff
@@ -49,8 +49,9 @@
     <dimen name="on_boarding_frag_image_size">250dp</dimen>
     <dimen name="spacing_none">0dp</dimen>
     <dimen name="nav_header_height">175dp</dimen>
+    <dimen name="card_height">175sp</dimen>
     <dimen name="fab_margin">16dp</dimen>
-    <dimen name="heading_text_size">25sp</dimen>
+    <dimen name="heading_text_size">25dp</dimen>
```

Comment:
Text sizes should be in sp.

Reply:
@jddeep I tried various implementations for this but couldn't find any other method for this, can you please help me a bit for the same.